.github/workflows/ci.yml

```diff
@@ -1087,6 +1087,103 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  # Runs tests on `tests/cli/partition_test`
+  partition_test:
+    name: Partitions Test (${{ matrix.test }}) on ${{ matrix.os }} for ${{ matrix.rust-target }}
+    runs-on: ${{ matrix.os }}
+    env:
+      FLUVIO_BIN: "~/bin/fluvio"
+      SERVER_LOG: fluvio=info
+    needs:
+      - build_primary_binaries
+      - config
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        rust-target:
+          - x86_64-unknown-linux-musl
+        test:
+          - multiple_partitions
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v3
+
+      # Download Artifacts from Development Build (This Commit)
+      - name: Download artifact - fluvio
+        uses: actions/download-artifact@v3
+        with:
+          name: fluvio-${{ matrix.rust-target }}
+          path: ~/bin
+
+      - name: Download artifact - fluvio-run
+        uses: actions/download-artifact@v3
+        with:
+          name: fluvio-run-${{ matrix.rust-target }}
+          path: ~/.fluvio/extensions
+
+      - name: Setup K3D
+        run: |
+          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=${{ env.K3D_VERSION }} bash
+          ./k8-util/cluster/reset-k3d.sh
+
+      - name: Wait 15s for K3D Reset
+        run: sleep 15
+
+      # This will load into ~/.fluvio/bin which will be in PATH
+      - name: Install Stable CLI and start Fluvio cluster
+        run: |
+          curl -fsS https://hub.infinyon.cloud/install/install.sh | bash
+          echo "~/.fluvio/bin" >> $GITHUB_PATH
+
+      - name: Check Directory Contents
+        run: ls -a ~/bin
+
+      - name: Set up Fluvio Binaries
+        run: |
+          chmod +x ~/bin/fluvio ~/.fluvio/extensions/fluvio-run
+          mv ~/bin/fluvio ~/.fluvio/bin/fluvio-dev
+          FLUVIO_BIN=~/.fluvio/bin/fluvio-dev
+          echo "FLUVIO_BIN=${FLUVIO_BIN}" >> $GITHUB_ENV
+
+      - name: Print version
+        run: fluvio version
+
+      - name: Start fluvio cluster
+        timeout-minutes: 10
+        run: fluvio cluster start --local
+
+      - name: Await Cluster Startup
+        run: sleep 15
+
+      - name: Stop Cluster to freeup SC Ports
```

Comment:
why shutdown after starting?

Reply:
check https://github.com/infinyon/fluvio/blob/master/.github/workflows/ci.yml for `local_cluster_testing`.   This should work if not it's bug